packages/fast-usdc/test/config-marshal.test.js

```diff
@@ -0,0 +1,45 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { Far } from '@endo/pass-style';
+import { mustMatch } from '@endo/patterns';
+import { AmountMath } from '@agoric/ertp';
+import { FastUSDCTermsShape } from '../src/type-guards.js';
+import {
+  fromLegible,
+  makeMarshalFromRecord,
+  toLegible,
+} from '../src/utils/config-marshal.js';
+
+const { keys } = Object;
+
+const testMatches = (t, specimen, pattern) => {
+  t.notThrows(() => mustMatch(specimen, pattern));
+};
+
+test('cross-vat configuration of Fast USDC terms', t => {
```

Comment:
this utility is neat. thanks for the tests; it made it click.

we'll want this to live in a more central place after FU settles. so not in the scope of this PR, but please do leave a comment to that effect or file an issue